vllm/model_executor/layers/quantization/fp8.py

```diff
@@ -306,11 +306,11 @@ def create_weights(self, layer: Module, num_experts: int, hidden_size: int,
         #   process_weights_after_loading()
         if self.quant_config.is_checkpoint_fp8_serialized:
             set_weight_attrs(w13_weight_scale, {
-                "is_fp8_scale": True,
+                "quant_method": "tensor",
```

Comment:
I think we should avoid this is possible

Would the VLLMParameters be able to support this?

Reply:
Yes, I just didnt want to bring it in yet so this was a quick workaround